compute_sdk/globus_compute_sdk/serialize/base.py

```diff
@@ -17,28 +18,59 @@ def __init_subclass__(cls):
         if len(cls.identifier) != IDENTIFIER_LENGTH:
             raise ValueError(f"Identifiers must be {IDENTIFIER_LENGTH} characters long")
 
+    #: The unique, 3-character string that identifies this strategy. The first two
+    #: characters are the effective identifier; the third must always be the newline
```

Comment:
```suggestion
    #: characters are the effective identifier, currently 2 numeric digits; the third must always be the newline
```

Reply:
I think that's more of an implementation detail than a requirement of the contract, no?